### PR TITLE
add README.md file

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Added README file

### CODE REVIEW

**Review by hinanshisuthar — COMMENTED** [review]
Looks good to me.
Suggestion: If you could provide a short description about the components like what they are and where are they mostly used, it would be easy for the user to actually make use of components better.

**Comment by junaidshaikh-js** [comment]
> Looks good to me. Suggestion: If you could provide a short description about the components like what they are and where are they mostly used, it would be easy for the user to actually make use of components better.

Yes, we could provide that, but it will be a very long readme file as there will be more components in the future. And we already stated that information in the documentation. So, I think it would be unnecessary to do so.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+
+## Installation Guide
+
+You can use the Hydrogen library by adding our stylesheet to your website. To get started add the following `link` element in your `head` element of the HTML file.
+
+```html
+<link rel="stylesheet" href="https://hydrogen-ui.netlify.app/style.css" />
+```
+
+Visit [docs](https://hydrogen-ui.netlify.app/) to learn more.
```
Comment (MaruthiPothuganti) [inline README.md]: Suggestion: directly link the docs URL instead of the homepage.

**Review by MaruthiPothuganti — COMMENTED** [review]
Change of **Visit Docs** _URL_

File: README.md (hunk 1 above)
Comment (junaidshaikh-js) [reply README.md]: Fixed!

**Pull request merged**